### DISCUSSION TITLE
Question 83: What are best practices for sensitive conservation status records? [IZ]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#875] [Q&A]
Description: How are the red balls triggered? What information is redacted as a result of red balls? Are they redacted on the IPT side? On the FMNH EMu website? How to deal with label images? How to deal with coordinates?

**Comment by rondlg** [comment]
Response: The red icon is triggered with PriRareAndEndangeredIndicator = “Yes”. For some collections (Inverts, Insects) having a value in the filedas Taxonomy record field GeoStatus_tab will trigger the word “listed” to appear in the summary string of the catalogue record.
Currently nothing is automatically excluded from the IPT data. We can program a note that indicates redacted fields that go out with the data.

**HOMEWORK**
- Each collection should:
1. Document how they think their redaction works now
2. Document how they would like it to work moving forward

[More information in warning icons](https://github.com/fieldmuseum/EMu-Documentation/wiki/admin-Warning-Icons)

**Reply by cmcmahanFMNH** [reply]
The same icon is triggered in this way for Fishes.  We indicate in Taxonomy when a taxon is ESA or CITES listed (IUCN does not apply).  We currently do not redact anything based on this indicator.  There are a handful of records will probably hide the lat/long or locality data for on a case-by-case basis.  We do not need an auto redaction for Fishes records.

**Comment by seanmkeogh** [comment]
**IZ**
1. _I think_ there is no redaction process for records published on the IPT. On the in house FMNH web portal the precise locality is redacted for sensitive species but geocoordinates are accessible regardless.
2. We are using the criteria outlined in Chapman 2020 (Current Best Practices for Generalizing Sensitive Species Occurrence Data) to identify sensitive taxa. So far no taxa meet the criteria needed to redact their locality information. Therefore no changes to our redaction procedure are needed at this time.

**Reply by chungkamei** [reply]
Amphibians & Reptiles historically and currently redacts precise locality information and geocoordinates. I'd like to leave it status quo and revisit in future if it needs to change.

**Comment by wyattgaswick** [comment]
Botany:
1) How things work now

   1a) We think that the red ball is triggered in Catalogue by the linked Taxonomy record. If it has Geog. Status->Status=”Threatened” or “Endangered”, and the listed Geog. Status->Region is a parent of the site linked in Catalogue, the red ball will appear. Unclear if the Red Data tab can trigger the ball.
	[EDIT: this seems to trigger regardless of whether the region is a parent; see IRN 3183784 which has the red ball even though it’s from Wisconsin, and the taxon is only listed in Maryland and Tennessee]

   1b) Redaction on website: precise locality is redacted (sites are only reported to state/province level), but the Lat/Long for the linked site appears regardless; all images show up.

   1c) Redaction in IPT: no automatic redaction. We have responsible for filtering things out ourselves but are not confident we have done a good job. Even the process for searching on PriRareAndEndangeredIndicator is unclear.

2) How we would like it to work

   2a) Red-ball should be triggered the same way, but should only affect Catalogue records in the specified geographical areas. Source data: ideally Axiell can supply red-list data and keep things updated (?) but if not we better figure out a way to do this ourselves. Solutions would be different between Botany sub-sections.

   2b) Redaction on website: Don’t publish images (unless we can come up with a fancy way to blur labels). Only publish sites to the County or PD3 level, with coordinates & uncertainty from the County or PD3 level record (If no PD3, use PD2 or next highest parent rank record). OK to publish elevation from the actual locality. [Alternately just leave off coordinates and elevation entirely if it’s too complex]. Include a note indicating that full info can be requested via email.

   2c) Redaction on IPT: same as on website. This may either require smart field reporting tools on the IT end, or careful procedure using the R&E flag on Botany’s end.

**Comment by mpdonovan2** [comment]
Geology:

Invertebrate Paleontology, Paleobotany, and Physical Geology do not redact anything currently. Locality information is displayed on photographed labels, making the information difficult to redact. In a situation where a specimen’s locality information should not be shared, we would not publish the record publicly. Vertebrate Paleontology would like to redact geographical data more specific than county and some accession data involving donor names.

**Comment by rondlg** [comment]
Fossil Verts request to redact precise locality information is documented here: https://github.com/fieldmuseum/Kiwi/issues/3
